debian/bareos-database-common.config.in

```diff
@@ -14,15 +14,20 @@ if [ -r @scriptdir@/bareos-config-lib.sh ]; then
 
         dbc_debug=1
         # prevent errors by using || true
-        dbc_dbname=`get_database_name bareos` || true
-        dbc_dbuser=`get_database_user bareos` || true
-
+        dbc_dbname=@db_name@ || true
+        dbc_dbuser=@db_user@ || true
         # convert Bareos databases types into dbconfig_dbtypes
-        dbc_dbtypes=""
-        DB_INSTALLED=`get_databases_installed`
-        if echo "$DB_INSTALLED" | grep -q postgresql; then dbc_dbtypes="${dbc_dbtypes}pgsql, "; fi
-        # remove trailing ,
-        dbc_dbtypes=`echo ${dbc_dbtypes} | sed 's/, *$//'`
+        dbc_dbtypes="pgsql"
+        # The Bareos catalog should be created with following settings:
+        # Encoding: SQL_ASCII
+        # Collate:  C
+        # Ctype: C
+        # While all recent versions of dbconfig support the parameter Encoding,
+        # Collate and Ctype are only supported in dbconfig >= 2.0.21.
+        # On older versions, these setting is ignored.
+        dbc_pgsql_createdb_encoding="SQL_ASCII"
+        dbc_pgsql_createdb_collate="C"
+        dbc_pgsql_createdb_ctype="C"
```

Comment:
I think, the values `dbc_pgsql_createdb_encoding`, `dbc_pgsql_createdb_collate` and `dbc_pgsql_createdb_ctype` are only evaluated in `postinst`.